docs/WHY.md

```diff
@@ -26,6 +26,78 @@ Mirascope addresses these pain points with a philosophy centered on transparency
     - **Type Safety**: Catch errors before runtime with our comprehensive type annotations.
     - **Intuitive API**: Our API is designed to be predictable and easy to learn, following Python best practices.
 
+## Comparison with Official SDKs
+
+Mirascope offers advantages over official SDKs like OpenAI's, particularly in terms of simplicity and developer experience. Let's compare the two:
+
+1. **Simplified Syntax**: 
+   
+OpenAI's official SDK:
+```python
+from openai import OpenAI
+
+client = OpenAI()
+completion = client.chat.completions.create(
+ model="gpt-3.5-turbo",
+ messages=[
+   {"role": "user", "content": "Hello world"}
+ ]
+)
+print(completion.choices[0].message.content)
+```
+
+Mirascope:
+```python
+from mirascope.core import openai, prompt_template
+
+@openai.call("gpt-3.5-turbo")
+@prompt_template("Hello world")
+def hello_world():
+   ...
+
+response = hello_world()
+print(response.content)
+```
+
+   Mirascope's approach streamlines the process using decorators, making it more concise.
+
+2. **Provider Agnosticism**: Mirascope allows you to easily switch between providers without changing your core logic. This is not possible with official SDKs, which are tied to specific providers.
+
+3. **Prompt Management**: Mirascope's `@prompt_template` decorator provides a way to manage prompts separately from logic. Official SDKs often mix prompts with API calls.
```

Comment:
same thing here, do you think a code example would help make this point more clear?

Reply:
i also wonder if this should be point (2) so it leads better into the point on being provider agnostic?